app/views/videos/show.html.haml

```diff
@@ -11,39 +11,50 @@
         %p= simple_format(@video.description)
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
+          = link_to("+ My Queue", queue_items_path(video_id: @video.id), method: :post, class: "btn btn-default")
+          / %a.btn.btn-default(href="") + My Queue
 
 %section.reviews.container
   .row
     .col-sm-10.col-sm-offset-1
-      %form
+      = form_for [@video, Review.new] do |f|
+        / %form
         %fieldset
           .form-group
             %label Rate this video
             .row
               .col-sm-3
-                %select.form-control(name="")
-                  %option(value="5") 5 Stars
-                  %option(value="4") 4 Stars
-                  %option(value="3") 3 Stars
-                  %option(value="2") 2 Stars
-                  %option(value="1") 1 Star
+                = f.select(:rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star"), number] }), {}, {class: "form-control"})
```

Comment:
For instance, this part `options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star"), number] })` could be refactored to `ApplicationHelper` and could be named :). You will see it later on anyway.